packages/contract-wrappers/src/contract_wrappers/contract_wrapper.ts

```diff
@@ -124,39 +112,8 @@ export abstract class ContractWrapper {
         const logWithDecodedArgs = abiDecoder.tryToDecodeLogOrNoop(log);
         return logWithDecodedArgs;
     }
-    protected async _getContractAbiAndAddressFromArtifactsAsync(
```

Comment:
Since we aren't reading addresses from a file anymore, I was able to remove this function and others like it. For `ContractWrappers` addresses are now known at instantiation time and are synchronous, which simplifies a lot of our code.